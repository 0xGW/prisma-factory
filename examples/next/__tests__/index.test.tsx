import { render, screen } from '@testing-library/react';
import Home from 'pages/index';
import React from 'react';
import renderer from 'react-test-renderer';

describe('Home page', () => {
  it('renders a heading', () => {
    render(<Home />);

    const heading = screen.getByRole('heading', {
      name: /Statistics/,
    });

    expect(heading).toBeInTheDocument();
  });

  it('renders Home page unchanged', () => {
    const tree = renderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
