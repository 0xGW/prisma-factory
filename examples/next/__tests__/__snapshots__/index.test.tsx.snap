// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders Home page unchanged 1`] = `
<div
  className="css-1tqjqq"
>
  <div
    className="chakra-stack css-owjkmg"
  >
    <h1
      className="chakra-heading css-0"
    >
      Statistics
    </h1>
    <section
      className="css-h40t3l"
    >
      <div
        className="css-wllvpf"
      >
        <div
          className="css-1wjac8z"
        />
      </div>
    </section>
  </div>
</div>
`;
